e2e/app/page/workspace-about-page.ts

```diff
@@ -33,4 +36,37 @@ export default class WorkspaceAboutPage extends AuthenticatedPage{
     return waitForAttributeEquality(page, {xpath: selector}, 'aria-selected', 'true');
   }
 
+  async findUserInCollaboratorList(username: string): Promise<WorkspaceAccessLevel> {
+    // At least one collab should eventually render, i.e. the current user.
+    const collabXPath = `//*[starts-with(@data-test-id,"workspaceUser-")]`;
+    await this.page.waitForXPath(collabXPath);
```

Comment:
`waitForXPath()` needs `{visible: true}` option here to ensure element is visible.

Reply:
Done